variables.tf

```diff
@@ -86,4 +80,18 @@ variable "tags" {
   description = "A mapping of tags to assign to all resources"
   type        = map(string)
   default     = {}
-}
\ No newline at end of file
+}
+
+variable "dlq_options" {
+  description = "Attributes to set in DLQ queue"
+  type = object(
+    {
+      max_message_receive_count  = optional(number)
```

Comment:
According to the ticket, this should not be optional
max_recieve_count: Integer, not optional, must be greater than 1

Reply:
Making  max_recieve_count as required will make regular use case to fail to create queue. It will expect dlq_options to set when we don't want to create one